ansible/roles/yum/tasks/main.yml

```diff
@@ -27,4 +27,4 @@
 - name: Add packages to versionlock
   shell: yum versionlock {{ item }}
   with_items: "{{ versionlock_items }}"
-  when: ({{ versionlock_items }} is defined)
+  when: (versionlock_items is defined)
```

Comment:
I think this is also now fixed, please let me know if your still having issues though. It might be a `ansible` version difference that is causing this to error for some people and not others. Please let me know what version you are on if you still experience issues.